src/protocolsupport/ProtocolSupport.java

```diff
@@ -127,7 +131,10 @@ public void onEnable() {
 		getServer().getPluginManager().registerEvents(new FeatureEmulation(), this);
 		getServer().getPluginManager().registerEvents(new ReloadCommandBlocker(), this);
 		getServer().getPluginManager().registerEvents(new MultiplePassengersRestrict(), this);
-		getServer().getScheduler().scheduleSyncDelayedTask(this, () -> (peserver = new PEProxyServer()).start());
+		getServer().getScheduler().scheduleSyncDelayedTask(this, () -> {
+			(peserver = new PEProxyServer()).start();
+		});
+		PECraftingManager.getInstance().registerRecipes();
```

Comment:
Should still do it in scheduled task (so all plugins registered their recipes), but before starting the server